packages/rancher-cis-benchmark/charts/templates/configmap.yaml

```diff
@@ -6,6 +6,8 @@ metadata:
 data:
   # Default ClusterScanProfiles per cluster provider type
   rke: "rke-profile-permissive"
+  rke2: "rke2-cis-1.5-profile-hardened"
```

Comment:
remove the hardened profile mapping from this file - here we maintain only one default profile per clusterProvider